device-types/Netgate/8300-Security-Gateway.yaml

```diff
@@ -0,0 +1,53 @@
+---
+manufacturer: Netgate
+model: 8300 Security Gateway
+slug: netgate-8300-security-gateway
+u_height: 1
+is_full_depth: false
```

Comment:
Hi thank you for your contribution.

To make sure we have the most complete data possible please add the following 'weight' and associated 'weight_unit" properties.